components/esb-tools/plugins/org.wso2.integrationstudio.esb.project/src/org/wso2/integrationstudio/esb/project/servlets/ConnectorDownloadServlet.java

```diff
@@ -89,15 +86,19 @@ protected IStatus run(IProgressMonitor monitor) {
                                         
                                         WizardDialogUtils.showSuccessMessage(connector.getAttributes().getOverview_name()
                                                 + " connector is downloaded Successfully.", DOWNLOADING_CONNECTOR_MSG, false);
+                                        
                                         return Status.OK_STATUS;
                                     }
                                 } catch (ConnectorException e) {
                                     WizardDialogUtils.showErrorMessage(e.getMessage(), DOWNLOADING_CONNECTOR_FAILED_MSG);
+                                    
                                 }
                                 monitor.worked(100);
                                 monitor.done();
                                 return Status.CANCEL_STATUS;
                             }
+                            
```

Comment:
shall we remove the additional new lines?

Reply:
fixed https://github.com/wso2/integration-studio/pull/1225/commits/32c29dcb1b7672b6d4b7e47378aa9862f28bf37c